resources/config.edn

```diff
@@ -0,0 +1,6 @@
+{:redis-uri #profile {:dev "redis://localhost:6379/"
+                      :prod #env REDIS_URL}
+ :host #profile {:dev "localhost"
+                 :prod "versions.deps.co"}
+ :port #profile {:dev 8080
+                 :prod 80}}
```

Comment:
I'm guessing that `:prod` should be set to 443 instead of 80, seeing as it's being used on https

Reply:
80 is fine, because it's running on Heroku who handle the SSL.